google/cloud/aiplatform/column.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Vertex AI tabular data types"""
+
+
+class data_types:
```

Comment:
I actually do wonder why we don't use an enum instead for this and the schema.

@sasha-gitg did you have a reason why?

Reply:
This class is used to generate Pipeline Components. AFAIK Pipeline Components do not support Python enums or Python enums nested in a dictionary.

Reply:
todo: track with a bug